packages/discovery/src/discovery/proxies/auto/GnosisSafe.ts

```diff
@@ -26,6 +27,35 @@ async function getMasterCopy(
   }
 }
 
+async function getModules(
+  provider: DiscoveryProvider,
+  address: EthereumAddress,
+  blockNumber: number,
+): Promise<EthereumAddress[]> {
+  const SENTINEL_MODULES = '0x0000000000000000000000000000000000000001'
```

Comment:
Would be nice if you could link to some documentation from Gnosis that explains that this address is a terminating value